benchmarking/smartseq2/RunBenchmarkingAnalysis.wdl

```diff
@@ -0,0 +1,78 @@
+import "run_combine_QCs.wdl" as CombineQC
+import "run_matrix_comparison.wdl" as CompareDataMatrix
+import "run_matrix_reproducibility.wdl" as ReproduciblityDataMatrix
+import "run_qc_metrics_analysis.wdl" as CompareQC
+import "run_confounding_factor_analysis.wdl" as ConfoundingDataMatrix
+
+workflow RunBenchmarkingAnalysis {
+  String scripts_dir
+  String base_datafile
+  String updated_datafile
+  String output_name
+  String gtf_file
+  String metadata_file
+  String metKeys
+  String base_metrics
+  String updated_metrics
```

Comment:
Could you double-check these types? I think you have a lot of files typed as `String`.